saylua/modules/forums/templates/main.html

```diff
@@ -4,7 +4,19 @@
   <link rel="stylesheet" type="text/css" href="{{ include_static('css/forums.min.css') }}">
 {% endblock %}
 {% block body %}
-<h1>Forums</h1>
+<h1>
+  <i class="fa fa-fw fa-comment" aria-hidden="true"></i>
```

Comment:
Maybe we should give these some alt / title text or something for accessibility.